cmd/kops/create.go

```diff
@@ -125,11 +126,19 @@ func RunCreate(f *util.Factory, out io.Writer, c *CreateOptions) error {
 	var sb bytes.Buffer
 	fmt.Fprintf(&sb, "\n")
 	for _, f := range c.Filenames {
-		contents, err := vfs.Context.ReadFile(f)
-		if err != nil {
-			return fmt.Errorf("error reading file %q: %v", f, err)
+		var contents []byte
+		if f == "-" {
```

Comment:
We will need to update the docs that we now support this.  Are you familiar with cobra and documentation?